src/Components/App.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+import Areas from '../Components/Areas';
+import  getHomeRepairs  from '../apiCalls';
+import NavBar from './NavBar';
+import Videos from '../Videos';
+import Projects from './Projects';
+import { Route, Switch } from 'react-router-dom'
+import '../App.css';
+
+class App extends Component {
+  constructor() {
+    super()
+    this.state = {
+      areas: ['kitchen', 'bathroom', 'bedroom', 'miscellaneous'],
+      homeRepairs: []
+    }
+  }
+
+  componentDidMount() {
+    getHomeRepairs()
+    .then(data => this.setState({homeRepairs: data}))
+  }
+
+  returnProjects = (projects, category) => {
+     if(projects.length > 0) {
```

Comment:
I believe that there is supposed to be a space between the if and its condition (e.g `if (projects.length > 0)`) That being said, if you haven't received feedback on this from an instructor, then I don't think its worth your time to go back and add a space after each if.